data/pages-content/jobs-board.md

```diff
@@ -3,32 +3,340 @@ title: Jobs Board
+
+<h6>Toronto, Canada </h6>
+
+<h5>About the role</h5>
+
+We are inviting applications for a Senior Backend Engineer to join our Applications team, leading the way in redefining code coverage. This position offers a unique chance to concentrate on the design, creation, and optimization of API features that form the backbone of our operations. The work you do will have a direct and significant impact on our wide user base and the efficiency of our systems. You'll be instrumental in building robust and scalable systems for data ingestion and processing, ensuring efficient handling of incoming data from multiple sources. Our team believes in the power of collaboration, and you'll be working alongside some of the most talented engineers in the industry in an environment that promotes learning. If you're ready to take on engaging challenges and make a meaningful contribution in the area of developer productivity, we're excited to hear from you!
+
+<h5>In this role you will</h5>
+- Collaborate with other engineers, product managers, and subject matter experts to deliver impactful outcomes to the organization.
+- Design and implement scalable, performant backend endpoints.
+- Design, build, and scale APIs for practical and robust systems.
+- Serve as a technical leader and mentor on one of Codecov’s core engineering teams, helping to unblock others and prioritize their tasks.
+- Frequently interact with other engineering, product, and business teams to help drive the future of Codecov’s products.
+</details>
+
+[Apply here](https://sentry.io/careers/4954424/) or contact: hr@sentry.io
+
+## [Numberly](https://numberly.com/en/)
+Numberly helps its clients collect, analyze and leverage their data across all channels to make marketing more relevant, effective and measurable. We are more than 200 engineers divided into teams with a human dimension and freedom of speech than enables everyone to have an impact and participate in the Group's development. Our sustained growth encourages us to constantly question our technical and organizational choices.
+
+Our challenges are international: 8 offices worldwide, and more than 30 nationalities in our main office in Paris.
+
+Our wide range of interconnected products and technical challenges are diverse and sophisticated. Our daily missions consist in processing thousands of requests per second, operating databases of several petabytes, automating our entire bare-metal infrastructure, and building the digital marketing interfaces of tomorrow.
+
+<details>
+  <summary><h4 id="numberly1-software-engineer">Software Engineer</h4></summary>
+  - thorough knowledge in a front-end or back-end stack
+  - join multiple product teams
+  - use exciting technologies (Python, Vue.js, Rust, Kafka, Kubernetes, ScyllaDB)
+  - build great platforms that leverage data
+  - work with Infrastructure Engineers and Data Engineers
+</details>
+
+[Apply here](https://nubr.ly/SE)
```

Comment:
I'm not sure if this is a correct link, I'm getting redirected to the main Numberly page.

The same for other Numberly links

Reply:
Yeah, I emailed to ask last night. Let's wait to hear back!

Reply:
ah, sorry, didn't check the email today

Reply:
NP! We can wait for them to reply before going live.